operations/dd-toc-guide.md

```diff
@@ -163,7 +163,7 @@ As an example, let's say a project is asserting their sub-projects have leadersh
 
 Another example, if the TOC member(s) is looking over the project's stated goals and objectives that have not changed since the project was accepted into the CNCF, and they are now applying to Graduate, the TOC member(s) will ask the project to clarify or provide additional information as to why the project hasn't iniatiated any changes and if they still feel those goals and objectives are accurate for the future of the project. The TOC member(s) will then summarize the response and record it in the PR under the corresponding criteria evaluation.
 
-It is expected that the TOC's evaluation of a project's completion of the criteria may reveal a mismatch in understanding or an unexpected implementation. Documenting the TOC evaluation in the Due Diligence PR provides the project, TAGs, community, adopters, and TOC with a point of reference to understand if the criteria are meeting the outcomes required of a project for a certain maturity level, or if compensating mechanisms that supplement or augment the criteria are in place that work best for that specific project.
+It is expected that the TOC's evaluation of a project's completion of the criteria may reveal a mismatch in understanding or an unexpected implementation. Documenting the TOC evaluation in the Due Diligence PR provides the project, TAGs, community, adopters, and TOC with a point of reference to understand if the criteria are meeting the outcomes required of a project for a certain maturity level, or if compensating mechanisms that supplement or augment the criteria are in place that work best for that specific project. We do not hold projects back from moving levels on factors outside the project’s control that are probabilistic occurrences because many factors can change at a moment's notice.
```

Comment:
> We do not hold projects back from moving levels on factors outside the project’s control that are probabilistic occurrences because many factors can change at a moment's notice.

This language is a bit confusing here. I understand it was taken verbatim from a private TOC thread, however, the context does not quite make sense here.

Reply:
Yes I was having a bit trouble to understand where this would fit, so I consulted @TheFoxAtWork and she suggested in here: https://github.com/cncf/toc/issues/1532#issuecomment-2649706362

Reply:
I understand .. it still is confusing here as it is out of context.

Reply:
I've removed it for now